content/influxdb/cloud-dedicated/query-data/troubleshoot.md

```diff
@@ -0,0 +1,194 @@
+| NotFound             | not found            | ✓             | 5         | NOT_FOUND        | The requested resource (action, data stream) wasn't found.        |
+| NotImplemented       | not implemented      | ✓             | 12        | UNIMPLEMENTED    | The RPC is not implemented.                                       |
+| RequestCanceled      | request canceled     | ✓             | 1         | CANCELLED        | The operation was cancelled (either by the client or the server). |
+| TooLarge             | request too large    | ✓             |           |                  |                                                                   |
+| Unauthorized         | unauthorized         | ✓             | 16        | UNAUTHENTICATED, UNAUTHORIZED | The client isn't authenticated or doesn't have permissions for the requested operation. |
+| Unavailable          | unavailable          | ✓             |           | UNAVAILABLE      | The server isn't available. May be emitted by the client for connectivity reasons.      |
+| Unknown              | internal error       | ✓             | 2         | UNKNOWN          | An unknown error. The default if no other error applies.          |
+| UnprocessableEntity  | unprocessable entity |               |           |                  |                                                                   |
+| EmptyValue           | empty value          |               |           |                  |                                                                   |
+| Forbidden            | forbidden            |               |           |                  |                                                                   |
+| TooManyRequests      | too many requests    |               |           |                  |                                                                   |
+| MethodNotAllowed     | method not allowed   |               |           |                  |                                                                   |
+| UpstreamServer       | upstream server      |               |           |                  |                                                                   |
+
+<!-- Reference: influxdb_iox/service_grpc_influxrpc/src/service#InfluxCode -->
+
+_For a list of gRPC codes that servers and clients may return, see [Status codes and their use in gRPC](https://grpc.github.io/grpc/core/md_doc_statuscodes.html) in the GRPC Core documentation._
+
+### Troubleshoot errors
+
+#### Internal Error: Received RST_STREAM
+
+**Example**:
+
+```sh
+Flight returned internal error, with message: Received RST_STREAM with error code 2. gRPC client debug context: UNKNOWN:Error received from peer ipv4:34.196.233.7:443 {grpc_message:"Received RST_STREAM with error code 2"}
+```
+
+**Potential reasons**:
+
+- The connection to the server was reset unexpectedly.
```

Comment:
I think this this can also happen if the query did not complete in the allotted time

Reply:
Meaning the server returns an error b/c the query didn't complete in the allotted time?